stateful/crd/crd.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: shareinfos.multishare.filestore.csi.storage.gke.io
+spec:
+  group: multishare.filestore.csi.storage.gke.io
+  names:
+    kind: ShareInfo
+    plural: shareinfos
+    singular: shareinfo
+    shortNames:
+    - si
+  scope: Cluster
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        # schema used for validation
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                storageClassTag:
+                  type: string
+                storageClassName:
+                  type: string
+                shareName:
+                  type: string
+                capacity:
+                  type: integer
+            status:
+              type: object
+              properties:
+                instanceHandle:
```

Comment:
please provide description of the fields with example of how the handle format looks like